code-studio/src/js/components/abuse_exclamation.jsx

```diff
@@ -8,91 +8,89 @@ window.dashboard = window.dashboard || {};
  * A big blue box with an exclamation mark on the left and our abuse text on
  * the right.
  */
```

Comment:
This comment should be moved down so it still applies to `AbuseExclamation`, rather than `AbuseError`.